src/main/scala/chisel/lib/dclib/package.scala

```diff
@@ -0,0 +1,14 @@
+package chisel.lib
+
+import chisel3._
+
+package object dclib {
+  def bin2grey(x : UInt) : UInt = (x >> 1.U) ^ x
```

Comment:
We have gray code conversion in the Chisel3 utils library now: https://github.com/chipsalliance/chisel3/blob/master/src/main/scala/chisel3/util/GrayCode.scala

Reply:
Ok I'll switch to the lib versions and test.  Also noticed that the checks were failing after submitting the request so will address those as well.

Reply:
Changed to chisel utils version of GrayCode in 910f360 below